site/README.md

```diff
@@ -44,6 +44,14 @@ The public-facing [blockprotocol.org](https://blockprotocol.org) website serves
     yarn dev
     ```
 
+### AWS configuration
+
+If you want to send verification codes to an email address, the following AWS environment variables have to be additionally configured:
+
+- `BP_AWS_REGION`: The region, eg. `us-east-1`
+- `BP_AWS_ACCESS_KEY_ID`: The AWS access key
+- `BP_AWS_SECRET_ACCESS_KEY`: The AWS secret access key
```

Comment:
Do we need these `BP_` prefixes?

Reply:
Unfortunately for now I think we do. Found out the hard way that Vercel reserves the AWS default environment variable names: https://vercel.com/docs/concepts/projects/environment-variables#reserved-environment-variables

Reply:
Ahhh, right. Fair enough - sorry for the pain this must've caused before discovering it!